src/output.h

```diff
@@ -26,6 +26,12 @@
 #undef Debug
 #endif
 
+#if defined __GNUC__ && (__GNUC__ < 4 || (__GNUC__ == 4 && __GNUC__ < 8))
+#define noreturn __attribute__((noreturn))
+#else
+#define noreturn [[noreturn]]
```

Comment:
Creating a macro called "noreturn" is dangerous. That could easily be a variable name or something that would get clobbered by the macro.

I would call it `EASYRPG_NORETURN` or `PLAYER_NORETURN` or something loud and macroey like this.